### PR TITLE
[extractor/heise] Fix extractor

### PR DESCRIPTION
**Pull request by coletdjnz**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

ADD DESCRIPTION HERE

Fixes https://github.com/yt-dlp/yt-dlp/issues/1520

For YouTube embed extraction, the site wraps the embed in what I assume is a custom "a-iframe" (alias of iframe?)
```
 <figure class="video video--fullwidth a-u-inline">
    
          <div class="video__yt-container">
            <a-iframe allowfullscreen class="video__iframe" frameborder="0" name="Youtube" needs-consent src="//www.youtube-nocookie.com/embed/6kmWbXleKW4" title="YouTube video player"></a-iframe>
          </div>
        
  </figure>
```

For one of the existing tests, the Kaltura extraction fails for the video (noted with a comment). I think this is an issue with the Kaltura extractor rather than Heise in this case. 

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [X] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [X] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [X] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [X] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [X] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

File: yt_dlp/extractor/heise.py [hunk 1]
```diff
@@ -127,20 +166,22 @@ def _make_kaltura_result(kaltura_url):
                 yt_urls, video_id, title, ie=YoutubeIE.ie_key())
 
         title = extract_title()
+        api_params = urllib.parse.parse_qs(
+            self._search_regex(r'\/videout\/feed\.json\?([^\']+)', webpage, 'feed params', default=None) or '')
```
Comment (pukkandan) [inline yt_dlp/extractor/heise.py:170]:
```suggestion
            self._search_regex(r'/videout/feed\.json\?([^\']+)', webpage, 'feed params', default=None) or '')
```

**Review by pukkandan — APPROVED** [review]

File: yt_dlp/extractor/heise.py [hunk 2]
```diff
@@ -23,6 +25,10 @@ class HeiseIE(InfoExtractor):
             'timestamp': 1512734959,
             'upload_date': '20171208',
             'description': 'md5:c934cbfb326c669c2bcabcbe3d3fcd20',
+            'uploader_id': '',
```
Comment (pukkandan) [inline yt_dlp/extractor/heise.py:28]: isnt this wrong?
```suggestion
```

File: yt_dlp/extractor/heise.py [hunk 3]
```diff
@@ -52,11 +74,16 @@ class HeiseIE(InfoExtractor):
             'description': 'md5:47e8ffb6c46d85c92c310a512d6db271',
             'timestamp': 1512470717,
             'upload_date': '20171205',
+            'duration': 786,
+            'view_count': int,
+            'uploader_id': '',
```
Comment (pukkandan) [inline yt_dlp/extractor/heise.py:79]: same

File: yt_dlp/extractor/heise.py (hunk 2 above)
Comment (coletdjnz) [reply yt_dlp/extractor/heise.py:28]: the Kaltura extractor is returning an empty string for these. the tests fail without it

Comment (pukkandan) [reply yt_dlp/extractor/heise.py:28]: better to have failing tests than wrong tests

**Pull request merged**